sdk/helper/keysutil/lock_manager.go

```diff
@@ -101,6 +101,24 @@ func (lm *LockManager) InvalidatePolicy(name string) {
 	}
 }
 
+func (lm *LockManager) RefreshCache(cacheSize int) error {
```

Comment:
Can we call this InitCache or something?  Refresh kind of implies you're refreshing the cached data somehow.

Reply:
Yes, that makes sense.